### PR TITLE
typo(ticdc): Fix spelling of 'true'

### LINKED ISSUE #6480: Typo in true
### What did you do?

https://github.com/pingcap/tidb/issues/36519

### What did you expect to see?

_No response_

### What did you see instead?

N/A

### Versions of the cluster

N/A

### PR DESCRIPTION
**Pull request by dveeden**
### What problem does this PR solve?

Issue Number: closes #6480

### What is changed and how it works?

See also https://github.com/pingcap/tidb/issues/36519

#### Tests 

 - No code

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by zhaoxinyu — APPROVED** [review]

File: tests/mq_protocol_tests/main.go [hunk 1]
```diff
@@ -113,7 +113,7 @@ func testMySQL() {
 
 func testMySQLWithCheckingOldValue() {
 	env := mysql.NewDockerEnv(*dockerComposeFile)
-	env.DockerComposeOperator.ExecEnv = []string{"GO_FAILPOINTS=github.com/pingcap/tiflow/cdc/sink/SimpleMySQLSinkTester=return(ture)"}
```
Comment (amyangfei) [inline tests/mq_protocol_tests/main.go:116]: Find `return(ture)` can also trigger failpoint, interesting.

**Review by amyangfei — APPROVED** [review]

**Comment by dveeden** [comment]
@amyangfei could you try to merge again?

File: tests/mq_protocol_tests/main.go (hunk 1 above)
Comment (lance6716) [reply tests/mq_protocol_tests/main.go:116]: `return` is enough for failpoint which doesn't use the value

**Pull request merged**